packages/create-snap/src/cli.ts

```diff
@@ -0,0 +1,37 @@
+import yargs from 'yargs';
+
+import builders from './builders';
+import { init } from './cmds';
+import { logError } from './utils';
+
+/**
+ * The main CLI entry point function. This processes the command line args, and
+ * runs the appropriate function.
+ *
+ * @param argv - The raw command line arguments, i.e., `process.argv`.
+ * @param initCommand - Optional specification for init command.
+ */
+export function cli(argv: string[], initCommand?: typeof init): void {
```

Comment:
This removes the need for `initCommand ?? init` further on.
```suggestion
export function cli(argv: string[], initCommand: typeof init = init): void {
```